test/scripts/bootstrap.sh

```diff
@@ -183,6 +183,9 @@ EOF
   AKS_MEMBER_1=$(az aks show -g $RESOURCE_GROUP -n $MEMBER_CLUSTER_1)
   AKS_MEMBER_1_NODE_RESOURCE_GROUP=$(echo ${AKS_MEMBER_1} | jq -r '. | .nodeResourceGroup')
 
+  echo "Assigning roles to hub kubelet identity on the MC_resourceGroup of the member cluster"
```

Comment:
I wonder why it's the kubelet identity?

Reply:
the agent is running on the node, so it's using node identity.